lumicks/pylake/kymotracker/kymoline.py

```diff
@@ -422,13 +422,23 @@ def save(self, filename, delimiter=";", sampling_width=None):
         """
         export_kymolinegroup_to_csv(filename, self._src, delimiter, sampling_width)
 
-    def fit_binding_times(self, n_components, *, exclude_ambiguous_dwells=True):
+    def fit_binding_times(
+        self, n_components, *, exclude_ambiguous_dwells=True, tol=None, max_iter=None
+    ):
         """Fit the distribution of bound dwelltimes to an exponential (mixture) model.
 
         Parameters
         ----------
         n_components : int
-            number of components in the model. Currently only values of {1, 2} are supported.
+            Number of components in the model. Currently only values of {1, 2} are supported.
+        exclude_ambiguous_dwells : bool
+            Determines whether to exclude dwelltimes which are not exactly determined. If `True`, tracks which
+            start in the first frame or end in the last frame of the kymograph are not used in the analysis,
+            since the exact start/stop times of the binding event are not definitively known.
+        tol : float
+            The tolerance for optimization convergence.
```

Comment:
This is stated too ambiguously, what tolerance is specified here?

Reply:
added a note that these are forwarded to `scipy.minimize()`